internal/security/proxy/service.go

```diff
@@ -105,7 +104,7 @@ func (s *Service) Init(cert CertificateLoader) error {
 	for _, name := range services {
 		params := Configuration.Clients[name]
```

Comment:
What happens if `name`  key not exists in that map?  Should we handle that corner case?

Reply:
add handle for this corner case.